README.rst

```diff
@@ -11,7 +11,7 @@ Enable custom checks for Odoo modules.
 +=======+====================================================================================================================================================================================================================================================================================================+======================================+
 | C7902 | Missing ./README.rst file. Template here: %s                                                                                                                                                                                                                                                       | missing-readme                       |
 +-------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------+
-| C8101 | Missing author required "%s" in manifest file                                                                                                                                                                                                                                                      | manifest-required-author             |
+| C8101 | One of the following authors must be present in manifest: %s                                                                                                                                                                                                                                       | manifest-required-author             |
```

Comment:
Shouldn't this be `manifest-required-authors` instead?

Reply:
I'd say no, because the lint wasn't renamed, only the parameter.

The reason for that is, the lint tells one required author is missing, while the parameter accepts multiple possible required authors. For that, I left the lint name singular and renamed the parameter to plural.

Of course, I'm open to suggestions regarding this.